config/ergw-c-node.config

```diff
@@ -74,7 +57,7 @@
 	   {[<<"apn02">>,<<"example">>,<<"net">>], [{vrf, sgi}]}
 	  ]},
 
-	 {teid, {3, 6}}, % {teid, {Prefix, Length}}
```

Comment:
@vkatsuba would {3,6} be the default if the config/ergw-c-node.config would be packaged via rebar?

Reply:
teh default (when otherwise unconfigured) comes from here https://github.com/travelping/ergw/blob/master/src/ergw_config.erl#L25

Reply:
Default seems to be {0,0}.
In ergw_config.erl

Reply:
yes, but since there "was" a teid-config in config/ergw-c-node.config - that would have been the new default, right?

Reply:
Yes, this is new default value

Reply:
This is a question of what defaults mean. I think it is the values used when there is nothing configured. In this case {3,6} is configured. So, not default. IMHO.

Reply:
> This is a question of what defaults mean. I think it is the values used when there is nothing configured. In this case {3,6} is configured. So, not default. IMHO.

Right. If `{teid, {Prefix, Length}}` is set, will be use value from config file, if not set, then will be `{0, 0}`.